### PR TITLE
Adding business cards!

### PR DESCRIPTION
**Pull request by JordanTranchina**
- SVG & PNG export
- HTML Spec export
- Sketch export

### CODE REVIEW

**Comment by antoinepetty** [comment]
I love both the back designs (slight preference to version B). The front designs don't need the "Teams:" etc bits imo because it's obvious from the email addresses. I prefer version B of the front.

**Comment by JordanTranchina** [comment]
@antoinepetty I addressed your request in the most recent commit. check it out and either ask for further changes or merge.

**Review by RealOrangeOne — CHANGES_REQUESTED** [review]
Looks like you've committed a `.DS_Store` file

**Comment by RealOrangeOne** [comment]
Side note: I'm not sure we need the teams address on these business cards. We have a very well defined process for teams signing up on the website, as the process is much more defined. 

The general, catch-all contact address for SR is `info@`, which might be a better fit to put on a business card like this.

@Tyler-Ward thoughts?

**Comment by Tyler-Ward** [comment]
Info at makes sense to have rather than teams, I forgot about that one when I came up with the email list.

It ti probably worth putting the files in a folder as well for when we add other designs to this repo as well. You might find it useful to add .DS_store to your global git ignore that way it wont need to add it to every .gitignore.

**Comment by JordanTranchina** [comment]
Cam yall summarize the changes you'd like to see? Hard to follow.

**Comment by RealOrangeOne** [comment]
@JordanTranchina 
- Remove the committed `.DS_Store` file
- Replace the `teams@` email address with `info@`

**Comment by antoinepetty** [comment]
> @JordanTranchina
> 
> * Remove the committed `.DS_Store` file
> * Replace the `teams@` email address with `info@`

I think it would make sense to have info@ at the top or bottom so that coreteam@ and trustees@ are grouped together. Not sure which you prefer?

**Comment by JordanTranchina** [comment]
Got it. I'll handle this and flip it back!

**Comment by JordanTranchina** [comment]
@Tyler-Ward @RealOrangeOne @antoinepetty 

Complete! Please request changes or approve the merge when you're ready :)

**Comment by antoinepetty** [comment]
Sorry for the slow progress on our responses @JordanTranchina ! 
@RealOrangeOne I'm thinking we need Info@ to be the top address because it's a catch-all - what are your thoughts on this? Are we aiming for the cards to mainly represent the core team and therefore the current iteration of having @ coreteam at the top is better?

**Comment by RealOrangeOne** [comment]
@antoinepetty I agree.

@JordanTranchina addresses should probably go `info@`, `trustees@`, `coreteam@`, just so we make sure the right addresses are used before other.

On the design front, these look really great, although I don't think we need multiple designs, I suspect having just the 1 will suffice. The front version D (looks like the latest) looks great! We can probably remove the other iterations. Thanks to `git` we can revert back if we change our minds. 

I also notice there's a zip file in the repo, is that an output from sketch or just a zip of all the designs? If it's the latter we probably don't need it in the repo. Simply the SVGs and sketch files should be enough.

Sorry it's taken so long. Once those changes are made I think we'll be basically ready to merge! :tada:

**Comment by JordanTranchina** [comment]
The zip file is a spec export like you may see from zeplin, but able to be opened via an html file. I can remove it in the future

**Comment by JordanTranchina** [comment]
Changes to address the above feedback is submitted! @Tyler-Ward @RealOrangeOne @antoinepetty

**Review by Tyler-Ward — APPROVED** [review]
looks good to me, lets get this merged
